webapp/TargetedMS/js/QCTrendPlotPanel.js

```diff
@@ -1881,9 +1899,233 @@ Ext4.define('LABKEY.targetedms.QCTrendPlotPanel', {
         };
         annotations.on("mouseover", function(){ return mouseOn(this, 3); });
         annotations.on("mouseout", function(){ return mouseOff(this); });
+
+        annotations.on("click", function (d) {
+            me.openAnnotationDialog(false, d).show();
+        });
+
+        // Add non-annotation markers with '+' shape
+        const addShape = function (size) {
+            var s = size / 2;
+            return 'M' + (-s) + ',0 L' + s + ',0 M0,' + (-s) + ' L0,' + s;
+        };
+
+
+        let nonAnnotationGroups = this.getSvgElForPlot(plot).selectAll("g.non-annotation-group").data(nonAnnotationsData)
```

Comment:
I understand non-annotation as a variable name for the dates without annotations, but it seems less descriptive when it comes to the UI elements. Consider `add-annotation` or similar since that's what the UI element is used for.